atomic_reactor/utils/operator.py

```diff
@@ -30,6 +30,95 @@ class NotOperatorCSV(Exception):
     """
 
 
+class NamedPullspec(object):
+    """
+    Pullspec with a name and description
+    """
+
+    _image_key = "image"
+
+    def __init__(self, data):
+        """
+        Initialize a NamedPullspec
+
+        :param data: Dict-like object in JSON/YAML data
+                     in which the name and image can be found
+        """
+        self.data = data
```

Comment:
I originally did that, but then I found it useful to access the data in [_related_image_env_pullspecs()](https://github.com/chmeliik/atomic-reactor/blob/c4aab84dccea5ced47ad433e7e9ef26a9e0866ff/atomic_reactor/utils/operator.py#L273). I could make `data` a property, but I didn't think it was too important since the class is meant to be internal to the module anyway